json-examples/external-ot-shelf.json

```diff
@@ -0,0 +1,1195 @@
+{
+  "ietf-network:networks": {
+    "network": [
+      {
+        "network-id": "WDM-Network-1",
+        "network-types": {
+          "ietf-te-topology:te-topology": {
+            "ietf-optical-impairment-topology:optical-impairment-topology": {}
+          }
+        },
+        "ietf-te-topology:te-topology-identifier": {
+          "topology-id": "WDM-Network-1"
+        },
+        "ietf-te-topology:te": {},
+        "ietf-optical-impairment-topology:otsi-group": [
+          {
+            "otsi-group-id": "Red OTSiG (Forward)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              }
+            ]
+          },
+          {
+            "otsi-group-id": "Red OTSiG (Reverse)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              }
+            ]
+          },
+          {
+            "otsi-group-id": "Green OTSiG (Forward)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              },
+              {
+                "otsi-carrier-id": 2
+              }
+            ]
+          },
+          {
+            "otsi-group-id": "Green OTSiG (Reverse)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              },
+              {
+                "otsi-carrier-id": 2
+              }
+            ]
+          }
+        ],
+        "node": [
+          {
+            "node-1": "WDM-TE-Node-1",
+            "ietf-te-topology:te-node-id": "192.0.2.1",
+            "ietf-te-topology:te": {},
+            "ietf-network-topology:termination-point": [
+              {
+                "tp-id": "1",
+                "ietf-te-topology:te-tp-id": 1,
+                "ietf-te-topology:te": {
+                  "inter-domain-plug-id": "AQ=="
+                }
+              },
+              {
+                "tp-id": "2",
+                "ietf-te-topology:te-tp-id": 2,
+                "ietf-te-topology:te": {
+                  "inter-domain-plug-id": "Ag=="
+                }
+              },
+              {
+                "tp-id": "3",
+                "ietf-te-topology:te-tp-id": 3,
+                "ietf-te-topology:te": {
+                  "inter-domain-plug-id": "Awo="
+                }
+              },
+              {
+                "tp-id": "23",
+                "ietf-te-topology:te-tp-id": 23
+              }
+            ],
+            "ietf-te-topology:tunnel-termination-point": [
+              {
+                "tunnel-tp-id": "AQ==",
+                "local-link-connectivities": {
+                  "is-allowed": false,
+                  "ietf-optical-impairment-topology:ttp-transceiver": [
+                    {
+                      "transponder-ref": 1,
+                      "transceiver-ref": 1
+                    }
+                  ],
+                  "local-link-connectivity": [
+                    {
+                      "link-tp-ref": 1,
```

Comment:
There are no link with termination point tp-id =1 in this network, it is in the other network, but the the relative path for leafref in ietf-te-topology.yang does not allow to get the idea from another network, it is in the same network.

      leaf link-tp-ref {
        type leafref {
          path "../../../../../nt:termination-point/nt:tp-id";
        }
        description
          "LTP.";
      }